openapi.yml

```
@@ -93,6 +93,76 @@ paths:
         '500':
           description: Internal Server Error
 
+  /users:
+    post: 
+      summary: user registration 
+      tags: 
+        - users
+      requestBody:
+        content:
+          application/json:
+            schema: 
+              $ref: '#/components/schemas/User'
+      responses:
+        '201':
+          description: complete registration
+          content:
+            application/json:
+              schema: 
+                $ref: '#/components/schemas/User'
+        '400':
+          description: Bad Request 
+        '500':
+          description: Internal Server Error
+
+
+  /login:
+    post: 
+      summary: user login
+      tags: 
+        - users
+      requestBody:
+        content:
+          application/json:
+            schema: 
+              $ref: '#/components/schemas/User'
+      responses:
+        '200':
```

特に返すものがないのであれば、`204 No Content` で良いかなと思います。
```suggestion
        '204':
```

---

トークンを返しているので、200がいいと思うのですが...いかがでしょうか？

---

すみません、指定行誤っていました:pray:
ログアウトの方ですね

---

aa6b5a2
修正しました！